frontend/src/components/QuestionsPage/QuestionsTop.tsx

```diff
@@ -35,18 +35,18 @@ const QuestionsTop: FunctionComponent<QuestionsTopProps> = ({ questions }) => (
           <BadgeGroup>
             {question.level &&
               question.level.map((level: string) => (
-                <Badge variant="primary" key={level}>
-                  {level}
-                </Badge>
+                <Fragment key={level}>
```

Comment:
Do we really need Fragment here and below?

Reply:
Yes, because I've added space after Badge and it counts as a separate 'item'